### PR TITLE
fear(server/setup): converting qb evidence

### LINKED ISSUE #1224: QB Police Evidence Conversion
**Is your feature request related to a problem? Please describe.**
locker numbers no longer have the items in them when changing evidence to use ox_inventory instead of qb-policejob

**Describe the solution you'd like**
Convert the police evidence stashes from qb db schema to ox table so that existing lockers contain the same items

**Describe alternatives you've considered**
N/A

**Additional context**
I'll probably PR this myself if someone else doesn't do it soon.

### PR DESCRIPTION
**Pull request by Manason**
fixes #1224

### CODE REVIEW

File: setup/convert.lua [hunk 1]
```diff
@@ -135,6 +135,26 @@ local function ConvertESX()
 end
 
 local function ConvertQB()
+
+	local function convertQbItems(items)
+		local converted = {}
+		local slot = 0
+		for _, v in pairs(items) do
+			if Items(v?.name) and v.amount then
```
Comment (Manason) [inline setup/convert.lua:143]: I kept the v.amount check from looking at the other examples in this file, but I'm not sure if it's possible for a qb item to have no amount.

**Comment by thelindat** [comment]
https://github.com/overextended/ox_inventory/blob/main/modules/bridge/qb/server.lua#L178

**Comment by thelindat** [comment]
> > https://github.com/overextended/ox_inventory/blob/main/modules/bridge/qb/server.lua#L178
> 
> We should extract a common function to convert items that isn't tied to a player and then refactor the convert code to call the bridge. I may do that in a separate PR.

Player is technically optional here; and used only for the sake of syncing physical accounts to items (money).

**Comment by Manason** [comment]
Calling the bridge conversion function to handle the conversion now.

File: setup/convert.lua [hunk 2]
```diff
@@ -303,6 +303,46 @@ local function ConvertQB()
...
+					oxEvidence[name][#oxEvidence[name]+1] = items[k]
+				end
+			else
+				oxEvidence[name] = items
+			end
+		end
+
+		for name, items in pairs(oxEvidence) do
+			count += 1
+			parameters[count] = { "INSERT INTO ox_inventory (owner, name, data) VALUES ('', :name, :data) ON DUPLICATE KEY UPDATE name = :name, data = :data", {
```
Comment (thelindat) [inline setup/convert.lua:332]:
```lua
"INSERT INTO ox_inventory (owner, name, data) VALUES ('', ?, ?) ON DUPLICATE KEY UPDATE name = VALUES(name), data = VALUES(data)"
```

**Review by thelindat — CHANGES_REQUESTED** [review]

**Pull request merged**